Source/WebCore/accessibility/AXTableHelpers.cpp

```diff
@@ -76,6 +76,24 @@ bool isTableRole(AccessibilityRole role)
     }
 }
 
+bool isARIAGridRow(Element& element)
+{
+    return hasRole(element, "row"_s);
+}
```

Comment:
The method name would be a bit more descriptive at each callsite if it was `hasRowRole`, since that's all this function does.